### PR TITLE
ENH: proposed change for returning pixels rather than channels

### PR DESCRIPTION
**Pull request by jrmlhermitte**
This is an API break but proposed change to resolve what was discussed at meeting (changing chip/chan to pixel number directly)

### CODE REVIEW

File: pygerm/client/__init__.py [hunk 1]
```diff
@@ -28,6 +32,11 @@ def payload2event(data):
     chip = word1 >> 27 & CHIP_BITMASK
     # chan addr
     chan = word1 >> 22 & CHAN_BITMASK
+
+    # convert to pixel
+    pixel = chip*32
```
Comment (mrakitin) [inline pygerm/client/__init__.py]: Do we still need this line?

Comment (jrmlhermitte) [reply pygerm/client/__init__.py]: no we don't. thanks!

File: pygerm/caproto.py [hunk 2]
```diff
@@ -198,6 +198,8 @@ def __init__(self, *, zclient,
                     fname = path / '{}.h5'.format(str(uuid.uuid4()))
                     with h5py.File(str(fname), 'w-') as fout:
                         g = fout.create_group('GeRM')
+                        # NOTE : this usage is now deprecated
+                        # Need to update if this class is used again
```
Comment (tacaswell) [inline pygerm/caproto.py]: Can you make this raise instead?

**Comment by tacaswell** [comment]
Do you plan to upgrade the IOC as well?

I would change the spec name to `binary_GeRM2` or something like that, we will need to keep both handlers around for as long as we want continue to read old data and have the better names on new data.

File: pygerm/caproto.py (hunk 2 above)
Comment (jrmlhermitte) [reply pygerm/caproto.py]: this should not have been necessary. removed

**Comment by jrmlhermitte** [comment]
what was changed was the reading, not the writing. i removed the change to `DATA_TYPES`.
This should not need a new handler number. This needs an hdf5 version test.

**Comment by tacaswell** [comment]
But the old data has UIDs for chip and chan which now fail?

**Comment by jrmlhermitte** [comment]
this is too long ago and the detector too custom. I will close this.
I don't think it's worth re-arranging unless it facilitates current work, or that the detector's API and behaviour is production ready.

**Pull request closed without merging**